docs/scenarios/parametric.md

```diff
@@ -327,51 +355,14 @@ cd utils/parametric
 
 Then you should have updated proto files. This script will generate weird files, you can ignore/delete these.
 
-## Implementation
-
-### Shared Interface
-
-#### HTTP
-
-An HTTP interface can be used instead of the GRPC. To view the interface run
-
-```
-./utils/scripts/parametric/run_reference_http.sh
-```
-
-and navigate to http://localhost:8000/docs. The OpenAPI schema can be downloaded at
-http://localhost:8000/openapi.json. The schema can be imported
-into [Postman](https://learning.postman.com/docs/integrations/available-integrations/working-with-openAPI/) or
-other tooling to assist in development.
-
-
-#### Legacy GRPC
-
-In order to achieve shared tests, we introduce a shared GRPC interface to the clients. Thus, each client need only implement the GRPC interface server and then these shared tests can be run against the library. The GRPC interface implements common APIs across the clients which provide the building blocks for test cases.
-
-```proto
-service APMClient {
-  rpc StartSpan(StartSpanArgs) returns (StartSpanReturn) {}
-  rpc FinishSpan(FinishSpanArgs) returns (FinishSpanReturn) {}
-  rpc SpanSetMeta(SpanSetMetaArgs) returns (SpanSetMetaReturn) {}
-  rpc SpanSetMetric(SpanSetMetricArgs) returns (SpanSetMetricReturn) {}
-  rpc SpanSetError(SpanSetErrorArgs) returns (SpanSetErrorReturn) {}
-  rpc InjectHeaders(InjectHeadersArgs) returns (InjectHeadersReturn) {}
-  rpc FlushSpans(FlushSpansArgs) returns (FlushSpansReturn) {}
-  rpc FlushTraceStats(FlushTraceStatsArgs) returns (FlushTraceStatsReturn) {}
-  rpc StopTracer(StopTracerArgs) returns (StopTracerReturn) {}
-}
-```
-
-
 ### Architecture
+Below is an overview of how the  testing architecture is structured:
+- Shared Tests in Python: We write shared test cases using Python's pytest framework. These tests are designed to be generic and interact with clients through the HTTP interface.
```

Comment:
```suggestion
- Shared Tests in Python: We write shared test cases using Python's pytest framework. These tests are designed to be generic and interact with the tracers through an HTTP interface.
```

Reply:
Can we maybe use the word "tracer" instead of "client" here and below? I feel like client is so obscure.

Reply:
In general, I just want users to understand that the methods written in test code to generate trace data make an outgoing http request through an Http client to an http server. On the http server the tracer is running and we carry out various operations with the tracer (e.g. generating spans, or adding tags) depending on which endpoints you hit and what values are passed in to the outgoing request.

From there we have two ways to check that the logic was carried out as expected, getting spans from the testagent (should provide example probably?) or looking at the value returned by the http server (example would also probably be helpful.)